magda-typescript-common/src/test/JsonTransformer.spec.ts

```diff
@@ -0,0 +1,153 @@
+import JsonTransformer, { JsonTransformerOptions } from "src/JsonTransformer";
+import ConnectorRecordId from "src/ConnectorRecordId";
+import { expect } from "chai";
+import "mocha";
+import AspectBuilder from "src/AspectBuilder";
+import * as fs from "fs";
+import { Record } from "src/generated/registry/api";
+
+describe("JsonTransformer", () => {
```

Comment:
What do you think if I keep 
  describe("organisationJsonToRecord")
but change
 it("should only revise the specific description",
and
  it("should not revise the non-specific descrition",
to
 it("should revise organisation record",
and
 it("should not revise organisation record",
respectively, because the currently unit test only test the method "organisationJsonToRecord"?
The latter is default behaviour.

In the future, we can add more tests on other methods.

Reply:
sounds good :). Maybe something like `organisation record should be the same if not revised` for the second one?

Reply:
done

Reply:
I also refactored the two concrete transformers in JsonTransformer.spec.ts.